am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -71,24 +71,30 @@ public void revokeResourceAccess(ExplicitAccessMetadata explicitAccessMetadata)
      * @return resourceJson or null
      */
     public FilterResourceResponse filterResource(String userId, String resourceId, JsonNode resourceJson) {
-        ExplicitAccessRecord explicitAccess = jdbi.withExtension(AccessManagementRepository.class,
+        List<ExplicitAccessRecord> explicitAccess = jdbi.withExtension(AccessManagementRepository.class,
             dao -> dao.getExplicitAccess(userId, resourceId));
 
-        if (explicitAccess == null) {
+        if (explicitAccess.isEmpty()) {
             return null;
         }
 
-        if (READ.isGranted(explicitAccess.getPermissions())) {
-            Map<String, Set<Permission>> attributePermissions = new ConcurrentHashMap<>();
-            attributePermissions.put("/", Permissions.fromSumOf(explicitAccess.getPermissions()));
+        Map<String, Set<Permission>> attributePermissions = new ConcurrentHashMap<>();
+
+        explicitAccess.forEach(explicitAccessRecord -> {
+            if (READ.isGranted(explicitAccessRecord.getPermissions())) {
```

Comment:
@yarpo do we still need this? Can we just list all permissions we found in DB without applying any additional logic to it?

Reply:
READ is _ONLY_ used to display data, not to filter permissions. All explicit permissions we store in our db for a resource should be displayed.